src/replication/model/registry.go

```diff
@@ -32,6 +32,7 @@ const (
 	RegistryTypeQuay             RegistryType = "quay"
 	RegistryTypeGitLab           RegistryType = "gitlab"
 	RegistryTypeDTR              RegistryType = "dtr"
+	RegistryTypeTcr              RegistryType = "tcr"
```

Comment:
I think it would be better to use tencent-tcr, which is more clear

Reply:
done